src/hotspot/cpu/x86/macroAssembler_x86.cpp

```diff
@@ -7210,7 +7215,6 @@ void MacroAssembler::kernel_crc32_avx512(Register crc, Register buf, Register le
   // For EVEX with VL and BW, provide a standard mask, VL = 128 will guide the merge
   // context for the registers used, where all instructions below are using 128-bit mode
   // On EVEX without VL and BW, these instructions will all be AVX.
-  lea(key, ExternalAddress(StubRoutines::x86::crc_table_avx512_addr()));
   notl(crc);
```

Comment:
We could do this not1(crc) in generate_updateBytesCRC32() thereby remove the need to do the double notl for crc32c.

Reply:
Moved the `notl(crc)` calls to `generate_updateBytesCRC32()` as requested.